docs/modules/ROOT/pages/cops_lint.adoc

```diff
@@ -3295,6 +3295,13 @@ challenging or verbose for no actual gain.
 Autocorrection is not supported because the position of `super` cannot be
 determined automatically.
 
+`Object` and `BasicObject` are excluded from this cop because of their
+stateless nature. However, sometimes you might want to exclude other parent
+classes from this cop, for example in the case of an abstract class that is
+not meant to be called with `super`. In those cases, you can use the
+`ExcludedParentClasses` option to specify which classes should be excluced
+*in addition to* `Object` and `BasicObject`.
```

Comment:
This document and examples are auto-generated based on the source code. Please update lib/rubocop/cop/lint/missing_super.rb directly.
There's no need to update the docs/modules/ROOT/pages/cops_lint.adoc doc as it will be generated at the time of release.

Reply:
Thank you @koic, I've amended the commit to keep the history clean.
I hope I did it right 😄